onnxscript/rewriter/pattern.py

```diff
@@ -729,15 +759,13 @@ def __init__(self, function) -> None:
 
     def get_replacement(
         self,
-        model,
-        vars: Sequence[Var],
         match_bindings: dict[str, ir.Value | Any] | None = None,
-    ) -> tuple[NodePattern | None, int | None]:
-        if match_bindings is None:
-            return None, None
+    ) -> ReplacementSubgraph:
         context = RewriterContext()
         new_values = self._function(context, **match_bindings)
-        return context.nodes
+        if not isinstance(new_values, Sequence):
+            new_values = [new_values]
```

Comment:
The new values are in fact new outputs of the subgraph?

Reply:
Right. Maybe new_outputs would be a better name